.github/workflows/test.yaml

```diff
@@ -1,15 +1,16 @@
 name: build
 on:
   push:
-    branches: ["master"]
   pull_request:
-    branches: ["master"]
+  # build weekly at 4:00 AM UTC
+  schedule:
+    - cron: '0 4 * * 1'
 jobs:
   test:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: ["2.7", "3.5", "3.6", "3.7", "3.8"]
+        python-version: ["2.7", "3.6", "3.7", "3.8"]
```

Comment:
Oh, also this is "obvious". We don't support 3.5 anymore, but it was still here in the test matrix.